### DISCUSSION TITLE
How can I get an Aggregate verifier?

### DISCUSSION THREAD

**Discussion by joaocosta9** [Web3Auth/Web3Auth#646] [Help]
Im building an app with my team, and I have the need to combine different attestation methods (login providers). It says in the docs that I can do this by creating an aggregate verifier. How can I achieve this?

Thank you

**Comment by rinzler99** [comment]
Hey, thanks for connecting! @tainguyentt can help here.

**Reply by tanguyenvn** [reply]
Hi @joaocosta9, for now we can only create aggregate verifiers manually. Let us know the details of aggregate verifier you need? We can chat via Telegram, my contact is tainguyentt
